doc/psyad/user_guide/implementation.rst

```diff
@@ -378,6 +378,35 @@ active then the loop statement is considered to be active. In this case:
           therefore avoid generating any loop-bound offset code in
           this case.
 
+If Statement
+------------
+
+When an if statement is found in the tangent-linear code, such as the
+following Fortran snippet:
+
+.. code-block:: fortran
+   
+   if (condition) then
+     ! content1
+   else
+     ! content2
+   end if
+
+1) the logical structure of the if is left unchanged, i.e. the "``if
```

Comment:
Looking at the output generated by Sphinx, I think having quotes ("") around things that are formatted using "``" looks odd. I suggest removing them.

Reply:
OK, I've changed the highlighting to `:code:` as well. The reason I added them is that I found it difficult to tell the difference between the fonts in the pdf version. I could still add single quotes around them if you agree and want me to make a better distinction.

Reply:
That looks fine.